index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <title>Document</title>
```

Comment:
- [ ] [Optional] You can change the document title here, you can add a custom title like: "My portfolio" or your name